policyengine_us/parameters/gov/states/ky/tax/income/deductions/standard.yaml

```diff
@@ -0,0 +1,15 @@
+description: Kentucky provides this standard deduction amount.
+values:
+  2021-01-01: 2_690
+  2022-01-01: 2_770
+metadata:
+  label: Kentucky standard deduction amount
+  period: year
+  unit: currency-USD
+  reference:
+  - title: Kentucky Revised Statutes, KRS Chapter 141.081 Optional standard deduction for individuals - Exception.
```

Comment:
```suggestion
  # Legal code specifies $1,700 in 1999, CPI-inflated since.
  - title: Kentucky Revised Statutes, KRS Chapter 141.081 Optional standard deduction for individuals - Exception.
```